server/app/controllers/api/v1/address_transactions_controller.rb

```diff
@@ -6,7 +6,7 @@ class AddressTransactionsController < ApplicationController
 
       def show
         address = Address.find_by!(address_hash: params[:id])
-        ckb_transactions = address.ckb_transactions.available.recent.page(@page).per(@page_size)
+        ckb_transactions = address.ckb_transactions.available.recent.distinct.page(@page).per(@page_size)
```

Comment:
Why here may has duplicate txs ?

Reply:
1. one transaction has payer and payee
2. output can be a change.